docs/source/03_tutorial/02_tutorial_template.md

```diff
@@ -50,28 +42,24 @@ wheel>=0.35, <0.37 # The reference implementation of the Python wheel packaging
 .. note::  If your project has ``conda`` dependencies, you can create a ``src/environment.yml`` file and list them there.
 ```
 
-### Add and remove project-specific dependencies
-
-The dependencies above may be sufficient for some projects, but for the spaceflights project, you need to add some extra requirements.
+The dependencies above may be sufficient for some projects, but for this tutorial you need to add some extra requirements. These will enable us to work with different data formats (including CSV, Excel and Parquet) and to visualise the pipeline.
 
-* In this tutorial, we work with different data formats including CSV, Excel and Parquet and want to visualise our pipeline so we will need to provide extra dependencies.
-* By running `kedro install` on a blank template we generate a new file at `src/requirements.in`. You can read more about the benefits of compiling dependencies [here](../04_kedro_project_setup/01_dependencies.md)
-* The most important point to learn here is that you should edit the `requirements.in` file going forward.
-
-Add the following requirements to your `src/requirements.in` lock file:
+Edit your `src/requirements.txt` file to include the following lines:
 
 ```text
 kedro[pandas.CSVDataSet, pandas.ExcelDataSet, pandas.ParquetDataSet]==0.17.6   # Specify optional Kedro dependencies
-kedro-viz==4.1.1                                                               # Visualise your pipelines
-openpyxl==3.0.9                                                                # Use modern Excel engine (will not be required in 0.18.0)
+kedro-viz~=4.0                                                                 # Visualise your pipelines
```

Comment:
Dumb question but will this install the latest version of `kedro-viz`?

Reply:
Yes 🙂  `kedro-viz~=4.0` effectively means `kedro-viz>=4.0, <5.0`. This is better than pinning a specific version since it means it will always install the latest.